src/styles/blocks/card.scss

```diff
@@ -0,0 +1,130 @@
+.main {
+  padding: 50px 40px;
+  display: grid;
+  grid-template-columns: repeat($column-count, 200px);
+  gap: 46px 48px;
+  justify-content: center;
+}
+
+@media (min-width: 488px) {
+  $column-count: 2;
+  .main {
+    grid-template-columns: repeat($column-count, 200px);
+  }
+}
+
+@media (min-width: 768px) {
+  $column-count: 3;
+  .main {
+    grid-template-columns: repeat($column-count, 200px);
+  }
+}
+
+@media (min-width: 1024px) {
+  $column-count: 4;
+  .main {
+    grid-template-columns: repeat($column-count, 200px);
+  }
+}
+
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  font-family: Roboto, serif;
+  transition-duration: $transition__duration;
```

Comment:
The variable `$transition__duration` is used here but is not defined in this file. Make sure this variable is defined in a file that is imported before this one, or define it at the top of this file.